### PR TITLE
Receive: add search transactions button

### PR DESCRIPTION
**Pull request by rating89us**
The "search transactions" button will open Transactions page and search all sent/received transactions of an address.

### CODE REVIEW

**Comment by xiphon** [comment]
Please use FontAwesome icon

**Comment by rating89us** [comment]
Do you want to use FontAwesome icon just for this new button, or for all three buttons (search, edit and copy to clipboard)?

**Comment by xiphon** [comment]
At least for the new button. Would love replacing all three buttons but think it is out of the current PR scope.

**Comment by selsta** [comment]
I would prefer if we keep the current icons, as they were custom designed for the GUI. Using FontAwesome for new icons is obviously fine.

**Comment by rating89us** [comment]
done

**Review by selsta — CHANGES_REQUESTED** [review]
You can use this code, it is simpler and looks closer like the other buttons. Also no changes to StandardButton required.

```
MoneroComponents.IconButton {
    fontAwesomeFallbackIcon: FontAwesome.searchPlus
    fontAwesomeFallbackSize: 22
    color: MoneroComponents.Style.defaultFontColor
    fontAwesomeFallbackOpacity: isOpenGL ? 0.5 : 0.8
    Layout.preferredWidth: 23
    Layout.preferredHeight: 21

    onClicked: doSearchInHistory(address)
}
```

**Review by selsta — APPROVED** [review]

File: pages/Receive.qml [hunk 1]
```diff
@@ -182,6 +182,17 @@ Rectangle {
                             height: 21
                             spacing: 10
 
+                            MoneroComponents.IconButton {
+                                fontAwesomeFallbackIcon: FontAwesome.searchPlus
+                                fontAwesomeFallbackSize: 22
+                                color: MoneroComponents.Style.defaultFontColor
+                                fontAwesomeFallbackOpacity: isOpenGL ? 0.5 : 0.8
```
Comment (selsta) [inline pages/Receive.qml:189]:
```suggestion
                                fontAwesomeFallbackOpacity: 0.5
```

#3491

**Review by selsta — APPROVED** [review]

**Pull request merged**